Sources/Runner/Commands/Runner.swift

```diff
@@ -24,6 +24,8 @@ func runDanger(logger: Logger) throws {
     //
     let dslJSONPath = dangerDSLURL.components(separatedBy: "danger://dsl/").last!
     logger.debug("Got URL for JSON: \(dslJSONPath)")
+    
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-default;
  File: Sources/Runner/Commands/Runner.swift;
  Line: 27;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)